website/source/api/jobs.html.md

```diff
@@ -231,6 +231,75 @@ $ curl \
 }
 ```
 
+## Parse Job
+
+This endpoint will parse an hcl jobspec and produce the equivalent json encoded
+job.
+
+| Method | Path                      | Produces                   |
+| ------ | ------------------------- | -------------------------- |
+| `POST` | `/v1/jobs/parse`          | `application/json`         |
+
```

Comment:
Add the standard blocking/acl section even though neither apply:

```
 | Blocking Queries | ACL Required               |
 | ---------------- | -------------------------- |
 | `No`             | `none`       |
```

Reply:
:heavy_check_mark: